include/diy/decomposition.hpp

```diff
@@ -327,36 +327,36 @@ decompose(int rank, const Assigner& assigner, const Creator& create)
       DivisionsVector     nhbr_coords(dim);
       Direction           dir, wrap_dir;
       bool                inbounds = true;
-      for (int i = 0; i < dim; ++i)
+      for (int k = 0; k < dim; ++k)
```

Comment:
Are these necessary? I thought Utkarsh dealt with this entire function body.

Reply:
I'll look again when rebasing; I may have been working on an older branch.

Reply:
It looks like he only changed the `i` above; this one still shadows the top-level `for` loop in this function.